pyi.py

```diff
@@ -1621,6 +1623,18 @@ def _check_if_expression(self, node: ast.expr) -> None:
         else:
             self.error(node, Y002)
 
+    def _check_for_Y066_violations(self, node: ast.If) -> None:
+        if (
+            isinstance(node.test, ast.Compare)
+            and ast.unparse(node.test).startswith("sys.version_info < ")
```

Comment:
You can use the backport we have for 3.8 to fix CI: https://github.com/PyCQA/flake8-pyi/blob/4aaa22d72cc754ecef1d07aaf4753050bb1915b2/pyi.py#L30

_But_ I'd rather we did this check without `ast.unparse`. It makes for a nice and simple test, so it suits the script in typeshed well, but I think it's likely to be much less efficient than just inspecting the structure of the AST without converting the AST into a string. It also feels slightly less principled and less type-safe to convert it into a string before inspecting the node